add: msm-cros-efs-loader

WeirdTreeThing opened the pull request.

WeirdTreeThing committed:
add: msm-cros-efs-loader

madonuko commented on anda/system/msm-cros-efs-loader/msm-cros-efs-loader.spec:
```diff
@@ -0,0 +1,35 @@
+install -Dm755 %{name}.sh %{buildroot}/usr/bin/%{name}
+install -Dm644 %SOURCE1 %{buildroot}/%{_unitdir}/msm-cros-efs-loader.service
+ 
+%post
+systemctl mask rmtfs.service
+%systemd_post msm-cros-efs-loader.service
```
what about `%preun` and `%postun`?

Also it might not be a good idea to run `systemctl mask` every single time the package updates?

madonuko commented on anda/system/msm-cros-efs-loader/msm-cros-efs-loader.spec:
```diff
@@ -0,0 +1,35 @@
+Source0:                %{url}/-/archive/v%{version}/msm-cros-efs-loader-v%{version}.tar.gz
+Source1:                msm-cros-efs-loader.service
+Requires:               rmtfs crossystem
+BuildArch:              noarch
```
Packager

Owen-sz committed:
Fix %post, add %preun and %postun

Preset file not made yet, and is not properly specified.

Signed-off-by: Owen Zimmerman <123591347+Owen-sz@users.noreply.github.com>

Owen-sz commented:
Don't merge until preset files are corrected.

%post, %preun, and %postun will not work yet.

Owen-sz committed:
Add packager

Signed-off-by: Owen Zimmerman <123591347+Owen-sz@users.noreply.github.com>

Owen-sz committed:
Fix: msm-cros-efs-loader.spec

https://github.com/Ultramarine-Linux/packages/pull/89

Will work when this is merged

Signed-off-by: Owen Zimmerman <123591347+Owen-sz@users.noreply.github.com>

Owen-sz commented:
Ready for merge

raboneko commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f40|[<img src="https://img.shields.io/github/pulls/detail/state/terrapkg/packages/2449">](https://github.com/terrapkg/packages/pull/2449)|
|✅|f41|[<img src="https://img.shields.io/github/pulls/detail/state/terrapkg/packages/2450">](https://github.com/terrapkg/packages/pull/2450)|
|✅|el10|[<img src="https://img.shields.io/github/pulls/detail/state/terrapkg/packages/2451">](https://github.com/terrapkg/packages/pull/2451)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11585644310) for details

<!--- Backport version: 9.3.0 -->
